debian/control

```diff
@@ -29,3 +29,14 @@ Description: high performance general RPC framework
  transparently, and makes it easier to build connected systems.
  .
  This package provides the gRPC C bindings.
+
+Package: libgrpc0-cil
+Architecture: any
+Section: cli-mono
+Depends: ${shlibs:Depends}, ${misc:Depends}
```

Comment:
This is the unexpanded control file. When I built the package, it expanded out to:
~~~
Depends: libc6 (>= 2.14), libgoogle-perftools4, libgrpc0, zlib1g (>= 1:1.1.4)
~~~